resources/views/adm/mship/feedback/new.blade.php

```diff
@@ -0,0 +1,173 @@
+@extends('adm.layout')
+
+@section('scripts')
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-sortable/0.9.13/jquery-sortable-min.js"
+            integrity="sha384-mwD0+87SDVjJjyfTMQHNVV+IyWDM38MhzdCFZ+SRefmD75v+M5K0R3naFNLnZf1L"
+            crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-datetimepicker/2.5.4/build/jquery.datetimepicker.full.min.js" integrity="sha384-8Lb23xW0dVl+HHrv90JF6PpwygXa7Z0zZIK9+RWorNDyubrG7Ppu7JJw32U8op0i" crossorigin="anonymous"></script>
```

Comment:
I had a brief look at this but from what I've seen you can't import JS from a URL using webpack unless you use third party modules, but since these modules aren't needed much I'd argue that it's simpler to just import them using script tags where needed. Please correct me if there is a simpler way to import external JS files with webpack though.